content/en/Platform Deep Dive/Assets/risk-advisories.md

```diff
@@ -23,9 +23,47 @@ When you [create](/platform-deep-dive/assets/#create-an-asset) or update an asse
 
 1. Once you’ve specified the [asset type](/platform-deep-dive/assets/#asset-types), select **Add Technology**, and start typing the technology name. We’ll show you a list of technologies that match your input.
 1. Select a technology with the exact version number.
+1. If your technology version is missing from the list, you can request to add it. Start typing the technology name, and then select **Add new version**. In the overlay that appears, submit a technology and its version, and we'll validate it for you.
+   {{%expand "Examples of ✅ valid technologies:" %}}
```

Comment:
Suggestion only. @ana-dashuk-cobalt , as you probably already know, I've included a note for these "expand" options in our Pentest Methodologies. I might add something like this line.

```
Select the <i style="font-size:x-large; color: #0047AB" class="fas fa-chevron-right"></i> key for specific examples.
```

Reply:
This change is a great improvement! 💙  I only wanna mention that the list of technology that's displayed here is also valid for the existing technologies. So not only are these technologies valid/invalid in terms of which to request/add by the user, instead it's only **valid** technologies that will ever be available in the dropdown. I wanted to say that in case you then want to rephrase it.

Reply:
@julisch94 We can add one more sentence to explain this:

![Screenshot 2022-11-30 at 21 11 10](https://user-images.githubusercontent.com/106706890/204898720-0a24832a-51f3-403e-8f82-b5e4686ea4aa.png)

Reply:
That's great thanks!